elements/pfe-modal/src/pfe-modal.js

```diff
@@ -185,6 +187,7 @@ class PfeModal extends PFElement {
     this._outer.setAttribute("hidden", true);
     // Return scrollability
     document.body.style.overflow = "auto";
+    document.body.style.position = "relative";
```

Comment:
We should be removing any and all styles that we add to `document.body` when the modal is closed. The current version leaves `overflow: auto` on the body tag. This update keeps the two additional styles on `document.body`. 

![Screen Shot 2020-08-24 at 8 45 56 AM](https://user-images.githubusercontent.com/330256/91046389-559fdb80-e5e6-11ea-82b1-7cfa13306339.png)

Reply:
That's a valid point. I'm open to other ideas about how to remove the jumpy scrollbar

Reply:
I removed position:fixed on the body, so right now the background is scrollable. But the focus is trapped inside the modal so it doesn't seem like an accessibility issue. I also am leaving in commented-out solution  about leveraging the scroll position in case someone wants to riff on this idea some more, but its imperfect. The window still jumps up a little bit when you close the modal the first time. The second time is perfect.

Reply:
(please provide suggestion on how to remove the two properties without removing the whole style attribute)